packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,100 @@
+| --------------------------------           | -----------   |
+| `interaction_code` |  The `interaction_code` is a one-time use, opaque code that the client can exchange for tokens using the Interaction Code grant type. This code enables a client to redeem a completed Identity Engine interaction for tokens without needing access to an authorization server’s session. |
+| `interaction_handle` | The `interaction_handle` is an opaque, immutable value that is provided by an Okta Authorization Server. The client can use the `interaction_handle` to interact with the Identity Engine component of the Okta org directly. The client is responsible for saving the `interaction_handle` and using it to hold the state of the transaction during remediation. Any public/confidential client that is configured to use the Interaction Code grant type can obtain an `interaction_handle`. If all the remediation steps are successfully performed, an `interaction_code` is returned as part of the success response.            |
+
+The following sequence of steps is a typical Interaction Code flow with the Identity Engine:
+
+<!--
+See http://www.plantuml.com/plantuml/uml/
+
+@startuml
+skinparam monochrome true
+actor "Resource Owner (User)" as user
+participant "Client" as client
+participant "Authorization Server (Okta)" as okta
+participant "Identity Engine (Okta)" as oie
+participant "Resource Server (Your App)" as app
+
+user -> client: Start auth with user info
+client -> client: Generate PKCE code verifier & challenge
+client -> okta: Authorization request w/ code_challenge, client ID, scopes, and user info
+okta -> okta: Remediation required
+okta -> client: Sends interaction_handle in response (for required interaction)
+user <-> client: Remedial interaction
+client <-> oie: Remedial interaction w/ interaction_handle
+note right: Possible multiple remedial steps required
+user <-> client: Remedial interaction
+client -> oie: Remedial interaction w/ interaction_handle
+oie -> oie: Remedial steps completed
+oie -> client: Send interaction_code in response
+client -> okta: Send interaction_code, client ID, code_verifier to /token
+okta -> okta: Evaluates PKCE code
+okta -> client: Access token (and optionally refresh token)
+client -> app: Request with access token
+app -> client: Response
+@enduml
+
+ -->
+
+![Interaction Code flow sequence diagram](/img/authorization/interaction-code-grant-flow.png)
+
+* The Interaction Code flow can start with minimal user information. For example, the user (resource owner) may only provide the client app with their username. Alternatively, the client can also begin the flow without any user information for a passwordless sign-in experience.
+
+* The client app generates the PKCE code verifier & code challenge.
+
+  > **Note:** To use the Interaction Code flow, both the client app and the Okta Authorization Server must have the Interaction Code grant type enabled in the Okta org.
```

Comment:
You already say that "Okta Authorization Servers must have the Interaction Code grant type enabled..." earlier on. Perhaps could combine these two and put the note earlier on, to cut down on repetition?

Reply:
I've decided to remove this whole sequence of steps explanation from this concept doc and move the content to the "Implement authorization by grant-type > Interaction Code" guide.

Reply:
ok, wfm.